src/shogun/classifier/mkl/MKLClassification.cpp

```diff
@@ -38,3 +38,16 @@ void CMKLClassification::init_training()
 	REQUIRE(m_labels->get_num_labels(), "Number of labels is zero.\n");
 	REQUIRE(m_labels->get_label_type() == LT_BINARY, "Labels must be binary.\n");
 }
+
+CMKLClassification* CMKLClassification::obtain_from_generic(CMachine* machine)
```

Comment:
@micmn wanna do some magic to create a static `cast` method for *all* Shogun classes?
Seems like you would know how to do that fast and then we could delete all the manual ones (and increase coverage) Thoughts?
@vigsterkr